scripts/pod_lib_lint.rb

```diff
@@ -83,6 +89,8 @@ def main(args)
       puts "Still working, running for #{number_of_times_checked / 60}min."
     end
   end
+
+  exit(pod_lint_status.exitstatus)
```

Comment:
`1` does not have an `exitstatus` member, so this is somewhat nonsensical given the initial value of `pod_lint_status`. Maybe assign it `$?.exitstatus` above?

Reply:
Right, that was the original intention. Thanks!